README.md

```diff
@@ -76,6 +76,7 @@ Configuration variables:
 - **departures** (*Required*): A list of routes and departure locations to watch
 - **route** (*Optional*): The name of the gtfs route
 - **stopid** (*Optional*): The stopid for the location you want etas for
+- **max_departures** (*Optional*): The maximum number of departures to retrieve. Defaults to 2. Specify 0 for all departures.
```

Comment:
Consider the following to make it more intuitive to users and also avoid adding multiple departures when users update?

- max_departures = 1, the default, one departure
- max_departures > 1, more than 1 departure
- max_departures = -1, all departures, typically -1 is used as a sentinel value for 'unlimited'
- max_departures = 0, invalid

Reply:
The default is currently 2, isn't it? I get a `Due at` and `Next bus` by default.

Reply:
Oops you are completely right. Sorry about that. I wasn't counting both 'due at' and 'next bus'. So I guess my comment should change to

- max_departures = 2, the default, due at + next up
- max_departures > 2, more than due at + next up
- max_departures = -1, all departures, typically -1 is used as a sentinel value for 'unlimited'
- max_departures = 0, invalid